hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/RouterServerUtil.java

```diff
@@ -52,7 +74,7 @@ private RouterServerUtil() {
   @Public
   @Unstable
   public static void logAndThrowException(String errMsg, Throwable t)
-      throws YarnException {
+          throws YarnException {
```

Comment:
The old indentation was correct, no?

Reply:
Thanks for your help reviewing the code, I will fix it.